test/functional/wicked_pdf_helper_assets_test.rb

```diff
@@ -6,18 +6,23 @@ class WickedPdfHelperAssetsTest < ActionView::TestCase
   include WickedPdfHelper::Assets
 
   if Rails::VERSION::MAJOR == 4
-    test 'wicked_pdf_asset_path should return an url when assets are served by an asset server' do
+    test 'wicked_pdf_asset_path should return a url when assets are served by an asset server' do
```

Comment:
Just more of a semantics thing, but it should probably be "an" url (see: http://itknowledgeexchange.techtarget.com/writing-for-business/which-is-correct-a-url-or-an-url/)